### DISCUSSION TITLE
"Pair Extraordinaire" shows one time as 2x and one time as 3x

### DISCUSSION THREAD

**Discussion by rugk** [community/community#63413] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

The “Pair Extraordinaire” badge shows in small under my profile as 2x achieved. Just below are the "new badges" and there it shows it 3x?

On https://github.com/rugk?tab=achievements

![screenshot](https://github.com/community/community/assets/11966684/2b27d99b-392c-450f-9cd7-58d1b2d5114a)

**Comment by radar-blip** [comment, marked as answer]
[Bug-Report](https://support.github.com/contact/bug-report)

**Reply by rugk** [reply]
Done: [Ticket #2293124](https://support.github.com/ticket/personal/0/2293124)

But why do I only find this now? The "give feedback" link brought me here, after all. I have searched for such a thing.

**Reply by radar-blip** [reply]
It seems the give feedback link is messed up too. 😕 

Your badge got fixed! 👍🏻

**Reply by rugk** [reply]
Apparently it was a cache thing, but well… as you can see yet alone from this issue that was the case for days hmm, should it be cached so long – and then show such an inconclusive response?

Response:
> Thanks for reaching out to GitHub Support. I have just triggered a recalculation of the Achievements data for your account and now see that the Pair Extraordinaire achievement now correctly shows the 3x badge in both places.
